Source/ablastr/fields/PoissonSolver.H

```diff
@@ -153,6 +160,23 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
                 {{ beta[0], beta[1], beta[2] }};
 #endif
 
+#if (defined(WARPX_USE_PSATD)  && defined(WARPX_DIM_3D))
+        // Use the Integrated Green Function solver (fft-based) on the coarsest level if it was selected
+        if(WarpX::poisson_solver_id==PoissonSolverAlgo::IntegratedGreenFunction && lev==0){
+            rho[lev]->mult(-PhysConst::ep0);
```

Comment:
With this we first divide and then multiply rho by `-eps0`, do we?
Can this be integrated above to avoid the necessary scaling for the IGF solver option?